cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -197,6 +197,13 @@ def create_xblock(parent_locator, user, category, display_name, boilerplate=None
 
         if display_name is not None:
             metadata['display_name'] = display_name
+        elif 'display_name' in metadata:
```

Comment:
Actually now I'm wondering why this is an `elif` rather than just standing on its own. What would be the problem doing

```python
if display_name is not None:
    metadata['display_name'] = display_name

metadata['display_name'] = _(metadata['display_name'])    # pylint: disable=translation-of-non-string
```
? Maybe I don't totally understand what you're doing here. I also don't think I understand fully why the `else` clause at L203 is doing.